File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.py
import sys
input=lambda:sys.stdin.readline().rstrip()
l=int(input())
r=int(input())
fine=r-l
if 1<=fine<=20:
    fine=100
elif 21<=fine<=30:
    fine=270
elif 31<=fine:
    fine=500
if fine<=0:
    print("Congratulations, you are within the speed limit!")
else:
    print(f"You are speeding and your fine is ${fine:.0f}.")

File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.c
#include <stdio.h>
int main(void) {
    int l, r, fine;
    scanf("%d", &l);
    scanf("%d", &r);
    fine = r - l;
    if (1 <= fine && fine <= 20) {
        fine = 100;
    } else if (21 <= fine && fine <= 30) {
        fine = 270;
    } else if (31 <= fine) {
        fine = 500;
    }
    if (fine <= 0) {
        printf("Congratulations, you are within the speed limit!");
    } else {
        printf("You are speeding and your fine is $%d.", fine);
    }
    return 0;
}